pages/common/grep.md

```diff
@@ -2,35 +2,36 @@
 
 > Matches patterns in input text.
 > Supports simple patterns and regular expressions.
+> More information: <https://man7.org/linux/man-pages/man1/grep.1.html>.
 
 - Search for a pattern within a file:
 
 `grep {{search_pattern}} {{path/to/file}}`
 
-- Search for an exact string:
+- Search for an exact ([F]ixed) string:
 
 `grep -F {{exact_string}} {{path/to/file}}`
 
 - Search for a pattern [R]ecursively in the current directory, showing matching line [n]umbers, [I]gnoring non-text files:
 
-`grep -RIn {{search_pattern}} .`
+`grep -RIn {{search_pattern}} {{.}}`
 
-- Use extended regular expressions (supporting `?`, `+`, `{}`, `()` and `|`), in case-insensitive mode:
+- Use [E]xtended regular expressions (supporting `?`, `+`, `{}`, `()` and `|`), in case-[i]nsensitive mode:
 
 `grep -Ei {{search_pattern}} {{path/to/file}}`
 
 - Print 3 lines of [C]ontext around, [B]efore, or [A]fter each match:
 
-`grep -{{C|B|A}} 3 {{search_pattern}} {{path/to/file}}`
+`grep -{{C|B|A}} {{3}} {{search_pattern}} {{path/to/file}}`
 
-- Print file name with the corresponding line number for each match:
+- Print file name [H]eaders with the corresponding line [n]umber for each match:
 
 `grep -Hn {{search_pattern}} {{path/to/file}}`
 
-- Use the standard input instead of a file:
+- Search for lines matching a pattern, printing [o]nly the matched text:
 
-`cat {{path/to/file}} | grep {{search_pattern}}`
+`grep -o {{search_pattern}} {{path/to/file}}`
 
-- In[v]ert match for excluding specific strings:
+- Search for lines that do not match (in[v]ert) a pattern:
 
 `grep -v {{search_pattern}}`
```

Comment:
Shall we add piping explicitly to this example? The way this is "worded", the command just hangs waiting for input. That would also allow us to demonstrate the piping behavior that is being removed above.

Reply:
👍🏻  https://github.com/tldr-pages/tldr/pull/5218/files#r594572492